scripts/collector/mysql/collect-data.sh

```diff
@@ -227,10 +227,9 @@ exit
 EOF
 done
 
-for x in $(grep -L DMA_SOURCE_ID output/*${V_FILE_TAG}.csv )
```

Comment:
@cofin i deleted this part as its injecting DMA_SOURCE_ID/MANUAL_ID to the csv files.
where they already have that.
Also i didn't find it yet but somewhere in the script the headers are converted to upper case.